app/services/food_service.rb

```diff
@@ -0,0 +1,32 @@
+class FoodService < BaseService
+
+  def self.get_details(food)
+    page_num = 1
+
+
+    response =  conn("https://api.nal.usda.gov").get("/fdc/v1/foods/search") do |f|
+      f.params["query"] = food
+      f.params["api_key"] = ENV['api_key']
+      f.params["pageSize"] = 200
+      # f.params["pageNumber"] = page_number
+
+      if !f.params["pageNumber"] == 5
+        page_num += 1
+      end
```

Comment:
No need for any pagination within the context of the stories. Love seeing you set your prams in the block. The API key should be set within the `conn` method in `BaseService`